app/scrapers/movie_scraper.rb

```diff
@@ -26,6 +26,10 @@ def scrape_tags(result)
     result['genre_ids'].map { |id| MoviedbClient::GENRES[id] }.uniq.compact
   end
 
+  def reliable?(result)
+    result['popularity'] > 4
```

Comment:
this value has been determined by try and error. seems to work for the james bond example.